tests/NexyCryptTest.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+namespace Nexy\NexyCrypt\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Nexy\NexyCrypt\NexyCrypt;
+use Nexy\NexyCrypt\Exception\AcmeApiException;
+
+class NexyCryptTest extends TestCase
+{
+    public $url = 'https://acme-staging.api.letsencrypt.org/';
+
+    public $domain = 'nexycrypt.esy.es';
+
+    public $keyPath = '/nexycrypt.private_key';
+
+    /** @test */
```

Comment:
Oh sorry, my fault.

The method must be **prefixed** by the `test` keywork, not suffixed.

Example here: https://github.com/nexylan/slack-bundle/blob/83652e1587a9655f958e575afc0af8cd4a184cb8/tests/DependencyInjection/ConfigurationTest.php#L14